html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML-Forms</title>
+</head>
+<body>
+  <h1>Coding Form</h1>
+  <form>
+    <label for="email">Email</label> <br>
+    <input id="email" type="text" name="email"> <br>
```

Comment:
you want to avoid using `br` tags, div tags are the proper way to break things onto multiple lines